src/main/java/no/nav/pto/veilarbportefolje/registrering/RegistreringService.java

```diff
@@ -1,16 +1,25 @@
 package no.nav.pto.veilarbportefolje.registrering;
 
 import no.nav.arbeid.soker.registrering.ArbeidssokerRegistrertEvent;
+import no.nav.pto.veilarbportefolje.domene.AktoerId;
+import no.nav.pto.veilarbportefolje.elastic.ElasticIndexer;
+
+import javax.inject.Inject;
 
 public class RegistreringService {
     private RegistreringRepository registreringRepository;
+    private ElasticIndexer elasticIndexer;
 
-    public RegistreringService(RegistreringRepository registreringRepository) {
+    @Inject
```

Comment:
hvorføøør må man har Inject av konstruktorn når jag selv new:ar upp bønnan??

Reply:
Hvordan vet Spring at `RegistreringRepository` (eller `RegistreringService` for den saks skyld), er en bønne?